examples/nlp/language_modeling/megatron_gpt_prune.py

```diff
@@ -45,9 +45,23 @@
     prune.num_attention_heads=null \
     prune.num_query_groups=null \
     prune.hidden_size=3072 \
-    export.save_path=llama3.1-8b-pruned.nemo
+    prune.num_layers=null \
+    export.save_path=llama3.1-8b-width-pruned.nemo
+```
+where tensor_model_parallel_size must be 1 because of the current prune API limitation
+
+Example usage to drop specific model layers (1-indexed):
+```
+python examples/nlp/language_modeling/megatron_gpt_prune.py \
+    model.restore_from_path=llama3.1-8b.nemo \
+    model.tensor_model_parallel_size=8 \
+    model.pipeline_model_parallel_size=1 \
+    trainer.num_nodes=1 \
+    trainer.precision=bf16 \
+    trainer.devices=8 \
+    'prune.drop_layers=[16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31]' \
+    export.save_path=llama3.1-8b-depth-pruned.nemo
```

Comment:
@kevalmorabia97 add an example where both depth and width is pruned?

Reply:
In some cases, we pruned both. Though one at a time, it should be okay to do it one-shot

Reply:
So you are suggesting I add the example script where we prune depth (cosine-similarity) and width both? 
Is it fine to add that in the TME documentation update though instead of here?

Reply:
better in both places as example usage is also provided here.

Reply:
addressed